docs/participate/03_treasury_proposal.md

```diff
@@ -11,6 +11,46 @@ The first step in applying for a Treasury grant is either to join the community
 This will help you get valuable community feedback throughout the process.
 It also gives Council members an open and transparent way to measure community sentiment.
 
+When submitting for discussion, it is best practice to include a proposal document.
+[A reference document](https://docs.google.com/document/d/1nz5GNjTL5inaI5s9djIki7DUp5zyPJV9dEI9K06kJ58/edit) can be found in the KILT Protocol discussions this can be used as a template for future proposals.
+
+## DID Sign
+
+The proposed document MUST be DID signed.
+This requires a DID that MUST be associated with the proposal.
+Therefore, it requires the proposer to have a DID.
+The signature provides integrity and accountability for the submitter, which can give more confidence to the community and council.
+
+The DID Signature should be done via [DIDSign](https://didsign.io/) a dApp which is built on KILT.
+The DIDSign doesn't have a database, nor store any data by the user.
+The following guide provides a [how to create a signature using a DID](https://kilt-protocol.org/files/How-to-Guide-DIDsign.pdf) with DIDSign.
+
+Once the document has been signed and a link can be added to the proposal so others can verify.
+This MUST include the document and signature.
+The method of storing the document and the signature can vary, however, we recommend using IPFS or an equivelant such as Crust.
+The document can be retrieved easily can be verified by anyone with a DID.
+
+Please include how to verify and download the necessary documents.
+The following is an example done by BOTLabs GmbH.
+
+```md
+## How to verify the document integrity
+
+The current version of the proposal document has been digitally signed with one of the DIDs that BOTLabs GmbH controls. To verify the signature:
+
+1. Download the PDF version of the Google Document linked above.
+
+2. Download the DID signature of the file from IPFS, with CID QmRcYyPcCKGDQno2m5qBSZq7dftoZKuwraF9C9M96rXR36 (e.g., ipfs.io).
+
+3. Visit didsign.io, and upload both the PDF file and the downloaded signature. The signature should verify correctly and link to the KILT tx in which the timestamp was generated.
+
+4. To verify the back-link from the DID to the account to which the funds must be sent if the Proposal is accepted, copy the account in the Proposal (15oXzySe6tjF2MumHfUodH8pFQWjy2hraRmXUJXXMKKY6p3F for quick reference) and paste it in w3n.id. The account must be linked to the same web3name that signed the proposal document.
```

Comment:
I would also remove this line, since we discussed the signer must not necessarily be linked to the payee.

Reply:
I removed the last step as it doesn't do what was intended. As we do not assume the signer and payee are the same and linked.

Reply:
No, but we are requiring that the **submitter** is linked to the DID signing the document. So we still require a link, but not between the receiver and the DID, right?
I mean we can also not require that, but I think that's what was decided. So you can't just take someone else's proposal and put it on chain, unless you sign it yourself.

Reply:
The removal of the line has happened. I removed the necessary linking.

Reply:
But we still need a link, right? Between the DID signer and the account submitter.

Reply:
How would we do this if someone doesn't want to link their account or they are not completely in control over it?

Reply:
I would say that someone submitting on behalf of someone else is an edge case and not the norm. Tips are suggested by other people, but treasury proposals should be submitted by the person asking for the money. But ok, we can keep this more relaxed and add stricter requirements if we need. Let's keep it like this for now.